File: PHP/test/TaxTableTest.php
<?php
namespace Test;

use Brick\Math\BigDecimal;
use PHPUnit\Framework\TestCase;
use Gov\TaxTable;

class TaxTableTest extends TestCase
{

    private $table;

    /** @before */
    public function setUpTaxTable()
    {
        $this->table = new TaxTable();
        $this->table->addProgression(BigDecimal::of(1000), 3);
    }

    /** @test */
    public function foo()
    {
        $this->assertTrue(BigDecimal::of(0)->isEqualTo(BigDecimal::of(1)));
    }

}

File: Java/src/test/java/gov/TaxTableTest.java
package gov;

import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.Matchers.is;

import java.math.BigDecimal;

import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

class TaxTableTest {

    private TaxTable table = new TaxTable();

    @BeforeEach
    void setUpTaxTable() {
        table.addProgression(new BigDecimal(1000), 3);
    }

    @Test
    void shouldFoo() {
        assertThat(true, is(false));
    }

}
